qa/common/rgw.sh

```diff
@@ -123,3 +123,17 @@ function rgw_ssl_init {
     popd
     rgw_add_ssl_global
 }
+
+
+function validate_rgw_cert_perm {
+    local TESTSCRIPT=/tmp/test_rados_put.sh
+    local RGWNODE=$(_first_x_node rgw-ssl)
+    cat << 'EOF' > $TESTSCRIPT
+set -ex
+trap 'echo "Result: NOT_OK"' ERR
+test $(stat -c'%U' /etc/ceph/rgw.pem) == "ceph"
```

Comment:
If the file does not exist, we get an unexpected error:

```
# test $(stat -c'%U' /non/existent/rgw.pem) == "root"
stat: cannot stat '/non/existent/rgw.pem': No such file or directory
bash: test: ==: unary operator expected
```

This can be avoided by putting the lval in double-quotes like so:

```
test "$(stat -c'%U' /non/existent/rgw.pem)" == "ceph"
stat: cannot stat '/non/existent/rgw.pem': No such file or directory
```

Also, could we put the `/etc/ceph/rgw.pem` in a variable and test that the file exists, something like:

```
RGW_PEM=/etc/ceph/rgw.pem
test -f "$RGW_PEM"
test "$(stat -c'%U' $RGW_PEM)" == "ceph"
test "$(stat -c'%G' $RGW_PEM)" == "ceph"
test "$(stat -c'%a' $RGW_PEM)" -eq 600
```

Reply:
thanks, will add this